convert_hf_to_gguf.py

```diff
@@ -295,7 +478,13 @@ def prepare_tensors(self):
                     bid = int(part)
                     break
 
-            for new_name, data_torch in (self.modify_tensors(data_torch, name, bid)):
+            for new_name, data_torch in (self._modify_tensors(data_torch, name, bid)):
+                # Some GPTQ models have empty bias tensors which are not in the model architecture.
+                # These tensors will cause tensor number check to fail, so we have to skip them.
+                if new_name.endswith(".bias") and np.all(LazyTorchTensor.to_eager(data_torch).numpy() == 0):
```

Comment:
```suggestion
                if new_name.endswith(".bias") and torch.all(data_torch == 0):
```

Lazy tensors should automatically become eager whenever an operation returns something else than a tensor (here, a `bool` from `torch.all`)

Reply:
@compilade Actually, this is a patch for some models we found. The structures between the model file and its arch don't match. If I remember correctly, it's some Qwen2 models.